### PR TITLE
Create application wrapper which reports to kibana when executed

### LINKED ISSUE #591: Log application usage
Make applications such as sched.summary and perturb.history log to kibana of any usage.

Easy solution would be to wrap the executables in a python wrapper which then makes a post to kibana before running the original executable.

### PR DESCRIPTION
**Pull request by ManInFez**
resolves #591

### CODE REVIEW

File: applications/wrapper.in [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
```
Comment (markusdregi) [inline applications/wrapper.in:1]: Should we put this in a ```__name__ == "__main__"```?

File: applications/wrapper.in [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+import os, pwd, socket, requests, subprocess, json, sys
+
+pw_entry = pwd.getpwuid(os.getuid())
+user = pw_entry.pw_name
+log_url = "http://devnull.statoil.no:4444"
```
Comment (markusdregi) [inline applications/wrapper.in]: Get this from `res.job_runner`

File: applications/wrapper.in [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            "komodo_release": os.getenv("KOMODO_RELEASE", "--------")
+}
+
+# Disabling proxies
+proxies = {
+    "http": None,
+    "https": None,
+}
+
+requests.post(log_url, timeout=3,
```
Comment (markusdregi) [inline applications/wrapper.in]: The wrapper should never fail, please wrap this in some exception handling...

File: applications/wrapper.in [hunk 4]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+import os, pwd, socket, requests, json, sys
+from job_runner import LOG_URL
+executable = "${EXEC}"
```
Comment (markusdregi) [inline applications/wrapper.in]: I think it would be cleanest to move this also into the main-block.

File: applications/wrapper.in [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+    requests.post(LOG_URL, timeout=3,
+                    headers={"Content-Type": "application/json"},
+                    data=json.dumps(payload), proxies=proxies)
+    
+if __name__ == "__main__":
+    try:
+        report()
+    except:
+        pass
```
Comment (markusdregi) [inline applications/wrapper.in]: Perhaps we should log something here?

File: applications/wrapper.in [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+                    headers={"Content-Type": "application/json"},
+                    data=json.dumps(payload), proxies=proxies)
+    
+if __name__ == "__main__":
+    executable = "${EXEC}"
+    try:
+        report(executable)
+    except Exception as e:
+        logging.warning("An error occured when logging application usage. "\
+            "Please contact the ERT-dev team at https://equinor.slack.com/messages/CDPJULWR4/")
```
Comment (markusdregi) [inline applications/wrapper.in]: This is a tiny bit too internal for an open source project ;) I suggest you remove the second line.

Comment (ManInFez) [reply applications/wrapper.in]: :+1:

**Review by markusdregi — APPROVED** [review]
Well done!

**Pull request merged**